infoschema/test/clustertablestest/tables_test.go

```diff
@@ -782,7 +782,7 @@ func TestFormatVersion(t *testing.T) {
 		"5.7.25-TiDB-v4.0.0-beta-446-g5268094af",
 		"5.7.25-TiDB-",
 		"5.7.25-TiDB-v4.0.0-TiDB-446"}
-	defaultRes := []string{"None", "8.0.18", "8.0.18-beta.1", "4.0.0-beta", "", "4.0.0-TiDB"}
+	defaultRes := []string{"None", "8.0.18", "8.0.18-beta.1", "4.0.0-beta-446-g5268094af", "", "4.0.0-TiDB-446"}
```

Comment:
What about adding two more versions than modifying existed cases?

Reply:
I didn't modify the existing test cases, I only modified the results.

| Input | Previous output | New output |
|---|:---|:---|
| `"5.7.25-TiDB-v4.0.0-beta-446-g5268094af"` | `"4.0.0-beta""` | `"4.0.0-beta-446-g5268094af"` |
| `"5.7.25-TiDB-v4.0.0-TiDB-446"` | `"4.0.0-TiDB"` | `"4.0.0-TiDB-446"` |

I've pushed a88e55e134c03f06921e66c0e678d40156bd71ca to refactor the test and make this a bit more obvious.